dapp/src/utils/connectors.js

```diff
@@ -42,9 +42,12 @@ export const walletlink = new WalletLinkConnector({
 })
 
 export function resetWalletConnector(connector) {
-  if (connector && connector instanceof WalletConnectConnector) {
-    connector.walletConnectProvider = undefined
-  }
+
```

Comment:
do not reformat unless prettier did that. to keep the diff clean.
to run prettier, do `npm run prettier` in the dapp directory.